tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op_gpu.cu.cc

```diff
@@ -411,6 +411,26 @@ class CuckooHashTableOfTensorsGpu final : public LookupInterface {
     return Status::OK();
   }
 
+  Status SaveToFileSystem(OpKernelContext* ctx, const string& filepath,
+                          const size_t buffer_size) {
+    int64 value_dim = value_shape_.dim_size(0);
+    cudaStream_t _stream;
+    CUDA_CHECK(cudaStreamCreate(&_stream));
+    return table_->save_to_file_system(ctx, value_dim, filepath, buffer_size,
```

Comment:
`return` will bypass the following `cudaStreamDestroy `

Reply:
fix